cozy-theme/nav.html

```diff
@@ -67,7 +68,7 @@
                         </a>
                     </li>
                 {%- endif %}
-              {%- endblock %}
+              {%- endblock %} -->
```

Comment:
Do not comment code, git is our memory, if we need it back, we can get it back from git revisions.
Future ourselves will never understand why this is commented out, help them by making clear action. And if you really want to remove this code but with revert possibility, just remove it in one single commit with a really explaining comment.

Reply:
I wanted to remove this code while explaining it can be possible for later use. So didn't want to remove it completely.

Reply:
Ne commente pas le code, git est notre mémoire, si nous avons finalement besoin de ce code, nous pourrons le récupérer grâce aux révisions git.
Nos moi futurs ne comprendront jamais pourquoi ce code est commenté, aidons les en faisant des actions claires. Et si tu penses que nous devrions supprimer ce code mais que nous pourrions en avoir besoin plus tard, retire-le dans un commit distinct avec un commentaire clair.